src/composer.rs

```diff
@@ -907,11 +907,11 @@ pub trait Composer: Sized + Index<Witness, Output = BlsScalar> {
         self.gate_mul(constraint)
     }
 
-    /// Adds a range-constraint gate that checks and constrains a
-    /// [`Witness`] to be inside of the range \[0,num_bits\].
+    /// Adds a range-constraint gate that checks and constrains a [`Witness`]
+    /// to be encoded in at most `num_bits`.
```

Comment:
Perhaps putting in the actual number range would be a good idea? Not like it was before, but something like `[0, 2^num_nbits[`

Reply:
I myself found the range in the comment slightly confusing as it gave me the impression that the number itself is in that range.
What this gate is actually doing though is constraining a number to be encoded in at most `num_bits` which will make it be in the range `[0, 2^num_bits[`. Do you want me to add this last part?

Reply:
That's what I thought it was doing. It's good to clarify this.